x-pack/docs/en/security/authentication/remote-clusters-privileges.asciidoc

```diff
@@ -33,8 +33,11 @@ On the remote cluster that contains the leader index, the {ccr} role requires
 the `read_ccr` cluster privilege, and `monitor` and `read` privileges on the
 leader index.
 
+NOTE: If requests will be authenticated with an <<security-api-create-api-key, API key>>, the API key
+requires the above privileges on the **local** cluster, instead of the remote.
+
 NOTE: If requests will be issued <<run-as-privilege,on behalf of other users>>,
-then the authenticating user must have the `run_as` privilege on the remote 
+then the authenticating user must have the `run_as` privilege on the remote
 cluster.
```

Comment:
I wonder whether we should mention that the API Key callout applies to the `run_as` privilege here as well. Or maybe it could get too complicated since the target user of `run_as` needs to be on the *remote* cluster?

Reply:
Hmm that's a good question. I think it's too complicated to be honest -- the flow at that point would be:

Define role on the remote
Unless it's an API key
Oh wait, if it's an API key using run-as then ... 

My preference would be to stick with the current formulation. Let me know if you feel strongly -- happy to try another tweak in a follow up.